$OpenBSD: patch-configure_ac,v 1.4 2021/11/23 13:04:34 sthen Exp $

Don't use gcc specific flags, because it causes various undefined
reference errors with ports-gcc>=6 when -lsupc++ is used. We could
use libesupc++ but there is not much incentive to do so.

Index: configure.ac
--- configure.ac.orig
+++ configure.ac
@@ -32,25 +32,6 @@ if test "$ac_prog_cxx_stdcxx" != cxx11; then
 fi
 
 LINKER="$CXX"
-dnl check whether we can link with gcc -lsupc++
-if test x$GCC = xyes && test x$GXX = xyes; then
-   dnl FreeBSD (at least up to 5.3) has a broken GCC, workaround it
-   AC_MSG_CHECKING([for working libsupc++])
-   save_CXX="$CXX"
-   save_LIBS="$LIBS"
-   CXX="$CC"
-   LIBS="$LIBS -lsupc++"
-   AC_LINK_IFELSE(
-      [AC_LANG_PROGRAM([struct test { }; void f() try { throw new test; } catch (...) { throw; }],[])],
-      [AC_MSG_RESULT(ok)
-       LINKER="$CC"],
-      [
-         AC_MSG_RESULT([no, making everything bigger and slower])
-         LIBS="$save_LIBS"
-      ]
-   )
-   CXX="$save_CXX"
-fi
 AC_SUBST(LINKER,[$LINKER])
 
 dnl# --------------------------------------------------------------------------
